BulletTrainClient.cs

```diff
@@ -295,7 +295,7 @@ private async Task<string> GetJSON(HttpMethod method, string url, string body =
             {
                 Console.WriteLine("\nHTTP Request Exception Caught!");
                 Console.WriteLine("Message :{0} ", e.Message);
-                return null;
+                return string.Empty;

```

Comment:
Check also[ this line](https://github.com/Flagsmith/flagsmith-dotnet-client/blob/d3b43b5378fdaf7fdcf4552983e029b850577023/BulletTrainClient.cs#L125) as `string.Empty`is deserialized as null.

Reply:
Thanks, didn't see that. Fixed.